lib/features/base/base_controller.dart

```diff
@@ -163,6 +164,15 @@ abstract class BaseController extends GetxController
     }
   }
 
+  void injectMailboxVisibilityBindings(Session? session, AccountId? accountId) {
+    try {
+      requireCapability(session!, accountId!, [CapabilityIdentifier.jmapTeamMailboxes]);
```

Comment:
srr for this problem if I understand wrong: `subcribed` need the capabilities of `team mailoxes`?

Reply:
Srr,  I re-read the story, it doesn't need it